src/components/Navigation.js

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+const linkStyles = {
+  padding: "0 2rem",
+  textDecoration: "none"
+};
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link style={linkStyles} to="/">
```

Comment:
this is an interesting way to add dynamic styling, but you're still probably better off using css for things like this.